test/trento/tags/policy_test.exs

```diff
@@ -0,0 +1,47 @@
+defmodule Trento.Tags.PolicyTest do
+  use ExUnit.Case
+
+  alias Trento.Tags.Policy
+  alias Trento.Tags.Tag
+  alias Trento.Users.User
+
+  test "should allow any action when the user has global ability" do
+    user = %User{abilities: [%{resource: "all", name: "all"}]}
+
+    Enum.each([:cluster, :host, :sap_system, :database], fn tag_resource ->
+      assert Policy.authorize(:add_tag, user, %{tag_resource: tag_resource, resource: Tag})
```

Comment:
you could add the `delete_tag` assertion one liner here as well

Reply:
done